subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -2565,10 +2598,57 @@ void radio_ar_configure(uint32_t nirk, void *irk, uint8_t flags)
 
 	NRF_AAR->ENABLE = (AAR_ENABLE_ENABLE_Enabled << AAR_ENABLE_ENABLE_Pos) &
 			  AAR_ENABLE_ENABLE_Msk;
+
+#if defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
+	aar_job.nirk = nirk;
+
+	/* Input, Resolvable Address Hash offset in the legacy or extended advertising PDU.
+	 * Radio packet pointer offset by 3 compared to legacy	NRF_AAR that took Radio packet
+	 * pointer value.
```

Comment:
```suggestion
	 * Radio packet pointer offset by 3 compared to legacy NRF_AAR that took Radio packet
	 * pointer value.
```

Could you add a reference for this? As someone who doesn't know what the legacy NRF_AAR is, I wouldn't know where to find more information.